docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -53,45 +55,59 @@ The steps in this guide require root privileges. Be sure to run the steps below
 
 ## Install Restic
 
-{{< content "applications/backups/install-restic-shortguide" >}}
+{{< content "install-restic-shortguide" >}}
 
 ## Create the Restic Repository
 
-{{< content "applications/backups/create-restic-repository-shortguide" >}}
+{{< content "create-restic-repository-shortguide" >}}
 
-## Set Up Automated Database Backups
+## Backup All Databases
```

Comment:
I wanted to separate the `Backup All Databases` content from the `Set Up Automated Database Backups` (replacing the `Run the backup script automatically` header) content. Those two headers are now both H2s

The goal of this was to create a heirarchy between the `Run the backup script automatically` header and `Cron` and `Systemd`, which were all H3s. I could have set `Cron` and `Systemd` as H4s, but I wanted to avoid going that far deep